script.py

```diff
@@ -101,6 +123,8 @@ def get_secret_values(self, secret_name):
             # https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_GetSecretValue.html
             raise e
 
+        self.logger.info(msg="Found secret", secret_name=secret_name)
```

Comment:
```suggestion
        self.logger.info(msg="Fecret found", secret_name=secret_name)
```

+you don't want to pass the k8s_namespace ?

Reply:
we don't have it there